detox/src/devices/ios/AppleSimUtils.js

```diff
@@ -70,8 +71,9 @@ class AppleSimUtils {
     const isBooted = await this.isBooted(udid);
```

Comment:
@d4vidi , you should probably change the signature of `boot(udid)`, and add to it `deviceLaunchArgs = ''` as a second argument.

AppleSimUtils.js is meant to be stateless, not depending on environment variables.